rulesengine_client/models.py

```diff
@@ -219,6 +227,22 @@ def protocol_applies(self, protocol):
             return True
         return self.protocol == protocol
 
+    def subdomain_applies(self, subdomain):
+        """Check to see whether the rule applies based on the query subdomain.
+
+        If the rule defines a canonicalized subdomain to match, we check to see
+        if the query subdomain matches.
+
+        :param subdomain: the query's subdomain
+        :type subdomain: str
+
+        :return: True if the rule defines no subdomain, or rule subdomain
+        matches param subdomain, otherwise False.
```

Comment:
Documentation pedantry: I find explaining just one side of the semantics of a returned `boolean` value to be sufficient, i.e., the `otherwise False` can be left off :)

Reply:
agreed, and I've updated.